src/index.js

```diff
@@ -593,11 +593,12 @@ const indy = {
     return JSON.parse(await IndySdk.proverGetCredentialsForProofReq(wh, JSON.stringify(proofRequest)))
   },
 
-  async proverSearchCredentialsForProofReq(wh: WalletHandle, proofRequest: ProofRequest, extraQuery: {}) {
+  async proverSearchCredentialsForProofReq(wh: WalletHandle, proofRequest: ProofRequest, extraQuery: {} | null) {
+    //The NodeJS IndySDK wrapper differs from the Java and iOS wrappers in this call--it allows extraQuery to be a wql query object or null.
     return await IndySdk.proverSearchCredentialsForProofReq(
       wh,
       JSON.stringify(proofRequest),
-      JSON.stringify(extraQuery)
+      JSON.stringify(extraQuery ? extraQuery : {})
```

Comment:
```suggestion
      JSON.stringify(extraQuery || {})
```

Reply:
Maybe even `JSON.stringify(extraQuery ?? {})` but I'm not sure if it'll be transpiled correctly with the current target 🤔

Reply:
That won't work if you pass null

Reply:
Nice suggestions here! I updated to @jakubkoci's suggestion, tested in RN and pushed.